### PR TITLE
#69 NotBlank refactored to IsBlank

### PR DESCRIPTION
**Pull request by Iprogrammerr**
As described in #69:

- Renamed NotBlank to IsBlank

- Negated it matchesSafely method

- Made similar changes to NotBlankTest

- Did not update README.md as it does not contain anything about this class

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#72](https://github.com/llorllale/cactoos-matchers/issues/72) is now in scope, role is `REV`

**Comment by dgroup** [comment]
@Iprogrammerr also, please add it to the readme.md

**Comment by Iprogrammerr** [comment]
@dgroup Refactored.

File: src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java [hunk 1]
```diff
@@ -28,53 +28,44 @@
...
         final Description desc = new StringDescription();
-        MatcherAssert.assertThat(
-            "must match a non-empty string",
-            new NotBlank().matchesSafely("text", desc),
-            new IsEqual<>(true)
-        );
-        MatcherAssert.assertThat(
-            "must describe itself in terms of the text being matched against",
-            desc.toString(),
+        new IsBlank().matchesSafely("text", desc);
```
Comment (dgroup) [inline src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java]: Use 
````java
@Rule
public ExpectedException 
````
in order to check the negative scenario as described here
https://github.com/llorllale/cactoos-matchers/blob/master/src/test/java/org/llorllale/cactoos/matchers/ScalarHasValueTest.java#L75

The using of `matchesSafely` was a bad idea and now we have the separate task to fix all tests regarding it.

**Comment by Iprogrammerr** [comment]
@dgroup Fixed.

File: src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java [hunk 2]
```diff
@@ -27,55 +27,53 @@
...
     @Test
     public void notBlank() {
-        MatcherAssert.assertThat(
-            "must match a non-empty string",
-            new NotBlank().matchesSafely(
-                "-.$%", new Description.NullDescription()
-            ),
-            new IsEqual<>(true)
-        );
+        this.exception.expect(AssertionError.class);
```
Comment (dgroup) [inline src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java]: @Iprogrammerr, could you please also check that the error message that it explains that the message isn't blank?

File: src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java [hunk 3]
```diff
@@ -27,55 +27,53 @@
...
     public void nonBlankMessage() {
-        final Description desc = new StringDescription();
-        MatcherAssert.assertThat(
-            "must match a non-empty string",
-            new NotBlank().matchesSafely("text", desc),
-            new IsEqual<>(true)
-        );
-        MatcherAssert.assertThat(
+        this.exception.expect(AssertionError.class);
+        this.exception.expectMessage(new StringContains("\"text\""));
```
Comment (dgroup) [inline src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java]: @Iprogrammerr, could you, please, check the whole line which contains the `text` (including `was: `) statement?

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Pull request merged**